goutubedl.go

```diff
@@ -604,7 +612,10 @@ func (result Result) DownloadWithOptions(
 	} else {
 		cmd.Args = append(cmd.Args, "--load-info", jsonTempPath)
 	}
-
+	//Force IPV4 Usage
+	if options.UseIPV4 {
```

Comment:
I think use `result.Options.UseIPV4` here and skip adding things to DownloadOptions. I guess most users want same options anyway?